### ISSUE TITLE
Links are not visibly recognizable  

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#18]
Links in the main content are distinguishable from surrounding text by color alone. This is not accessible to people who are unable to perceive color differences, and is a violation of WCAG 2.0 at Level A ([Success Criterion 1.3.3](www.w3.org/TR/WCAG20/#content-structure-separation-understanding)). 

On the [Accessibility site](http://uw.edu/accessibility), you added (at our request) a 1px solid border-bottom as a solution that is more subtle than text-decoration: underline. However, this is _awfully_ subtle, barely visible for some people and we've received some complaints about it. 

Preferred solution: 
1. Push an easier-to-see solution to the Accessibility site (either a darker border-bottom color, or 2px instead of 1px or both). 
2. Get some sort of solution - either subtle or less subtle - pushed universally

**Comment by nwinkelbauer** [comment]
This was addressed with https://github.com/uweb/uw-2014/pull/47

**Comment by terrill** [comment]
This was only partially addressed. The "Preferred solution" in my original issues included two recommendations. The first of these has been addressed, but I don't think the second has.  All pages other than those with class="site-accessibility" still use color alone to identify links, which is a violation of WCAG 2.0 as noted above.  

Unless the author of page content has gone wild with links, I think underlining links really won't cause an aesthetic problem. Also, it probably only needs to be added to paragraphs within the main content. For example consider the [Discover the UW](http://www.washington.edu/about/) page: 

- There's a single link "UW Today" at the end of the first paragraph within div#main_content, which is inaccessible to people who can't distinguish the difference in color. It should be underlined so everyone can easily spot it. 
-  There are additional lists of links under various subheadings, still within div#main_content, but these don't require links because it's easy to tell they're links based on the context.
